.unimportedrc.json

```diff
@@ -1,4 +1,14 @@
 {
   "ignoreUnresolved": ["i18n"],
-  "entry": ["*.config.ts", "*.config.cjs", "pages/**/*", "scripts/**/*", "i18n/index.ts"]
+  "entry": [
+    "sentry.client.config.ts",
+    "sentry.edge.config.ts",
+    "sentry.server.config.ts",
+    "*.config.cjs",
+    "pages/**/*",
+    "scripts/**/*",
+    "i18n/index.ts"
+  ],
+  "ignoreUnused": ["next", "react", "react-dom"],
+  "ignoreUnimported": ["wagmi.config.ts", "cypress.config.ts"]
```

Comment:
please, revert

Reply:
Both `@wagmi/cli` and `cypress` are bringing dependencies which for some reason are conflicting with `unimported`. This is the output I am getting
![image](https://github.com/exactly/app/assets/9066191/d4a1f96a-c09f-400e-b941-4ff0459bf034)

Updating with `npx unimported -u` didn't help. Opted to ignore those 2 entry files, for which I had to narrow `*.config.ts`

Reply:
@jgalat what changed about this? every config file is an entry point

Reply:
We are now using `@wagmi/cli` for generating the abi types. Its configuration is on `wagmi.config.ts`, which imports stuff from that same package. I suspect these deps are only used in the CLI and that's why they are showing as unimported by the library types. 

Then for `cypress.config.ts`, we had to introduce `@cypress/webpack-preprocessor` (which brings `webpack`, on first line). The issue we had with this is was that e2e tests were failing due to cypress being unable to recognize an optional chaining operator in the new `viem` dependencies. Changing the target from the tsconfig didn't solve this, which lead me to this option. Here is an example of running the tests without it (failing with due to using `_` in numbers)
![image](https://github.com/exactly/app/assets/9066191/a4067858-9ec6-418c-b9b7-c5d474c8cf00)

Due to all of this and being unable to just ignore those imports with `unimported -u`, i decided to just remove them from the entry points list